### ISSUE TITLE
Issue entity_encoding

### ISSUE THREAD

**Issue by TamokiLoi** [tinymce/tinymce#7036]
I have a code with a character encoding, i used entity_encoding: 'raw', but when i submit code still have character. Please help me resolved isssue...

`<!--mobile will replace 3 comments below with appropriate data--> <!--TITLE--> <!--AUTHOR--> <!--DATETIME--> <!--editor-content--> <!--editor-content--><p><img src="./1626312232880_files/original.jpeg" alt="Logo.jpg" width="200" height="112" data-mce-src="https://cdn.icondo.asia/images/50974d32/f0e6/48cc/bea2/20a18b03e091/original.jpeg"></p><p><span></span><br></p><p><span>Dear Visitor / Contractor, welcome to The Topiary. Kindly comply with the by-laws as follows: <br></span></p><ul><li><span>Parking hours for visitors is from 0700 hours to 0000 hours. No parking is allowed after 12:00 midnight</span></li><li><span>No overnight parking for visitors withouth prior approval from The Management.</span></li><li><span>Parking hours for contractors is strictly from 0900hours to 1700hours. All contractor vehicles will have to leave the estate by 1700hours.</span></li><li><span>Non-display of the parking slip will be deemed as unauthorized parking.</span></li><li><span>Vehicles will be wheel clamped for any non-compliance to the parking rules at The Topiary.</span></li><li><span>A release fee of S$200.00 shall be payable in cash for the release of the immobilized vehicle.</span></li><li><span>The Management shall not be liable for any compensaiton or claims for theft, damages and loss of contents to the parked vehicle.<br><br><strong>By Order  The Management</strong><br><br>It is the sole responsibility of the visitor/contractor to ensure the input particulars are correctly filled in, also consent to the collection of your personal data and its use by The Management and its agents. The Management is entitled to retain the information for audit purposes.<br></span></li></ul><!--/editor-content--><p><br></p><!--jquery--><p><script src="./1626312232880_files/jquery.min.js" type="mce-no/type" data-mce-src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script><script src="./1626312232880_files/hammer.js" type="mce-no/type" data-mce-src="https://cdnjs.cloudflare.com/ajax/libs/hammer.js/2.0.8/hammer.js"></script></p><!--custom script--><p><script type="mce-no/type">`

![Capture](https://user-images.githubusercontent.com/27203631/126602843-7b0b5b0a-8403-4018-a45c-685e5e0c5520.PNG)
